app/controllers/two_factor_authentication/add_methods_controller.rb

```diff
@@ -0,0 +1,43 @@
+module TwoFactorAuthentication
+  class AddMethodsController < ApplicationController
+    before_action :confirm_two_factor_authenticated
+
+    def index
+      @two_factor_add_methods_form = TwoFactorAddMethodsForm.new
+      @presenter = two_factor_options_presenter(after_sign_in_path_for(current_user))
+    end
+
+    def create
+      @two_factor_add_methods_form = TwoFactorAddMethodsForm.new
+      result = @two_factor_add_methods_form.submit(two_factor_add_methods_form_params)
+
```

Comment:
Please add an analytics event.

Reply:
It's coming. I'm just trying to get the specs working at the moment.